packages/web-react/src/components/RadioField/RadioField.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useStyleProps } from '../../hooks/styleProps';
+import { SpiritRadioFieldProps } from '../../types';
+import { useRadioFieldStyleProps } from './useRadioFieldStyleProps';
+
+export const RadioField = (props: SpiritRadioFieldProps): JSX.Element => {
+  const { classProps, props: modifiedProps } = useRadioFieldStyleProps(props);
+  const { id, label, value, isDisabled, isChecked, onChange, ...restProps } = modifiedProps;
+  const { styleProps, props: otherProps } = useStyleProps(restProps);
+
+  return (
+    <div {...styleProps} className={classNames(classProps.root, styleProps.className)}>
```

Comment:
This implementation is different from HTML (#404) and Twig (#405). Why is that? 🤔 If it's really necessary, we will need to make `id` required.

Reply:
right point, fixed implementation..